app/_locales/hn/messages.json

```diff
@@ -0,0 +1,819 @@
+  },
+  "blockiesIdenticon": {
+    "message": "ब्लॉकीज पहचान का उपयोग करें"
+  },
+  "borrowDharma": {
+    "message": "धर्मा (बीटा) से / के साथ उधार लें"
+  },
+  "builtInCalifornia": {
+    "message": "मेटामास्क कैलिफ़ोर्निया में डिज़ाइन और बनाया गया है।"
+  },
+  "buy": {
+    "message": "खरीदें"
+  },
+  "buyCoinbase": {
+    "message": "कॉनबेस पर खरीदें"
+  },
+  "buyCoinbaseExplainer": {
+    "message": "बिल्टकोइन, एथरेम और लाइटकोइन खरीदने और बेचने के लिए दुनिया का सबसे लोकप्रिय तरीका Coinbase है।"
+  },
+  "cancel": {
+    "message": "रद्द करें"
+  },
+  "classicInterface": {
+    "message": "क्लासिक इंटरफ़ेस का उपयोग क"
+  },
+  "clickCopy": {
+    "message": "कॉपी करने के लिए क्लिक करें"
+  },
+  "confirm": {
+    "message": "पुष्टि करें"
+  },
+  "confirmContract": {
+    "message": "अनुबंध की पुष्टि करें"
+  },
+  "confirmPassword": {
+    "message": "पासवर्ड की पुष्टि करें"
+  },
+  "confirmTransaction": {
+    "message": "लेनदेन की पुष्टि करें"
+  },
+  "continue": {
+    "message": "जारी रखें"
+  },
+  "continueToCoinbase": {
+    "message": "कॉ्ोनबेस को ब्हेजना जारी रखें"
+  },
+  "contractDeployment": {
+    "message": "अनुबंध परिनियोजन व तैनाती"
+  },
+  "conversionProgress": {
+    "message": "रूपांतरण प्रगति में हे।"
+  },
+  "copiedButton": {
+    "message": "कॉपी किया गया"
+  },
+  "copiedClipboard": {
+    "message": "क्लिपबोर्ड पर कॉपी किया गया"
+  },
+  "copiedExclamation": {
+    "message": "कॉपी कर दिया गया!"
+  },
+  "copiedSafe": {
+    "message": "मैंने इसे कहीं सुरक्षित कॉपी कर दिया है"
+  },
+  "copy": {
+    "message": "कॉपी / प्रतिलिपि कर्रे"
+  },
+  "copyToClipboard": {
+    "message": "क्लिपबोर्ड पर कॉपी करें"
+  },
+  "copyButton": {
+    "message": " कॉपी / प्रतिलिपि कर्रे [बत्तन] "
+  },
+  "copyPrivateKey": {
+    "message": "यह आपकी निजी कुंजी है (कॉपी करने के लिए क्लिक करें)।"
+  },
+  "create": {
+    "message": "बनाएं"
+  },
+  "createAccount": {
+    "message": "खाता बनाएं"
+  },
+  "createDen": {
+    "message": "डेन बनाएं"
+  },
+  "crypto": {
+    "message": "क्रिप्टो",
+    "description": "एक्सचेंज टाइप (क्रिप्टोक्यूचरस)"
+  },
+  "currentConversion": {
+    "message": "वर्तमान रूपांतरण"
+  },
+  "currentNetwork": {
+    "message": "वर्तमान नेटवर्क"
+  },
+  "customGas": {
+    "message": "अनुकूलित करें गैस"
+  },
+  "customize": {
+    "message": "अनुकूलित करें"
+  },
+  "customRPC": {
+    "message": "कस्टम RPC"
+  },
+  "decimalsMustZerotoTen": {
+    "message": "दशमलव कम से कम 0 होनी चाहिए, और 36 से अधिक नहीं होनी चाहिए।"
+  },
+  "decimal": {
+    "message": "दशमलव परिशुद्धता"
+  }, 
+  "defaultNetwork": {
+    "message": "ईथर लेनदेन के लिए डिफ़ॉल्ट नेटवर्क मुख्य नेट है।"
+  },
+  "denExplainer": {
+    "message": "आपका डेन मेटामास्क के भीतर आपका पासवर्ड-एन्क्रिप्टेड स्टोरेज है।"
+  },
+  "deposit": {
+    "message": "जमा"
+  },
+  "depositBTC": {
+    "message": "नीचे दिए गए पते पर अपना बीटीसी जमा करें:"
+  },
+  "depositCoin": {
+    "message": "नीचे दिए गए पते पर अपना $ 1 जमा करें",
```

Comment:
`$1` is a variable and must be written exactly this way

Reply:
Oh! Sorry that escaped my conscience the other night!
`variable` correction enroute ( :